### PR TITLE
feat: add sonar cloud properties for better control

### PR DESCRIPTION
**Pull request by sawood14012**
https://issues.redhat.com/browse/RHTAPREL-713
Adding Sonar Cloud Properties File for better control

### CODE REVIEW

File: .sonarcloud.properties [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Path to sources
+sonar.sources=.
+sonar.exclusions=**/*test*, **/vendor/**, **/zz_generated.*, **/*.yaml,
```
[the remark this answers is not in the document]
Comment (sawood14012) [reply .sonarcloud.properties]: vendor removed

**Review by johnbieren — APPROVED** [review]

**Pull request merged**